File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.VettingService.Models/get-courtesy-file_result.ts


import { IGetCourtesyFile_Result } from './iget-courtesy-file_result';

export class GetCourtesyFile_Result implements IGetCourtesyFile_Result {
  
	public FileID: number = 0;
	public FileVersion: number = 0;
	public FileName: string = "";
	public FileSize: number = 0;
	public FileHash: number[] = null;
	public FileContent: number[] = null;
  
	public ErrorMessages: string[] = null;
}

File: GTTS/INL.VettingService.Models/GetCourtesyFile_Result.cs
using System;
using System.Collections.Generic;
using System.Text;
using INL.Services.Models;

namespace INL.VettingService.Models
{
    public class GetCourtesyFile_Result : BaseResult, IGetCourtesyFile_Result
    {
        public long FileID { get; set; }
        public int FileVersion { get; set; }
        public string FileName { get; set; }
        public int FileSize { get; set; }
        public byte[] FileHash { get; set; }
        public byte[] FileContent { get; set; }
    }
}
